apps/devhub/views.py

```diff
@@ -677,8 +679,15 @@ def standalone_upload_detail(request, uuid):
 @dev_required
 @json_view
 def upload_detail_for_addon(request, addon_id, addon, uuid):
-    upload = get_object_or_404(FileUpload, uuid=uuid)
-    return json_upload_detail(request, upload, addon_slug=addon.slug)
+    with statsd.timer('devhub.upload_detail_for_addon'):
+        try:
+            upload = get_object_or_404(FileUpload, uuid=uuid)
+            return json_upload_detail(request, upload, addon_slug=addon.slug)
+        except:
+            statsd.incr('devhub.upload_detail_for_addon_error')
```

Comment:
This looks useful. Can you also add a counters for 1) every case, regardless of success/error and 2) a count of every success. That will allow us to make a graph of what percentage of every upload is a failure.